doc/zmq.md

```diff
@@ -102,7 +104,17 @@ is structured as the following based on the type of message:
 
 Where the 8-byte uints correspond to the mempool sequence number.
 
-These options can also be provided in bitcoin.conf.
+`rawtx`: notifies when a transaction is added to mempool. The body of the message is the raw transaction in hexadecimal format.
+
+`hashtx`: notifies when a transaction is added to mempool. The body of the message is structured in the following way:
+
+    hashtx <32-byte transaction hash>
```

Comment:
`hashtx` is not contained in the payload. The messages are ZMQ multipart messages with three parts. The first part is the topic (here `hashtx`), the second part is the 32-byte transaction hash, and the last part is a sequence number (to detect lost messages; this is already documented). 

I'd personally prefer a documentation similar to the following:
```
| hashtx | <32-byte transaction hash in Little Endian> | <uint32 sequence number in Little Endian>
```

Reply:
I think it would be good to explicitly note that the 32-byte hashes are in Little Endian and not in the Big Endian format that the RPC interface and e.g. block explorers use to display transaction and block hashes.

Reply:
Done.